### ISSUE TITLE
Not booting

### ISSUE THREAD

**Issue by iWhacko** [byte4RR4Y/rock-3a-kali#1]
The device turns on, shows green light. then blue light which stays on. does not flash.
After 30minutes nothing has happened, screen stays off as well. Keyboard does not light up so I do not think it's booting at all

**Comment by byte4RR4Y** [comment]
without an own device its hard to build it

**Comment by iWhacko** [comment]
yes, I understand. Got any documents or webpages showing how to make a build? 

Else I will try to install the kali packages on an armbian build. See if that works

**Comment by byte4RR4Y** [comment]
i also work on a debian testing build for the rock 4 se. got a lot of projects, but i will try my best
